lib/compress/zstdmt_compress.c

```diff
@@ -329,10 +341,18 @@ void ZSTDMT_compressChunk(void* jobDescription)
         if (ZSTD_isError(initError)) { job->cSize = initError; goto _endJob; }
     } else {  /* srcStart points at reloaded section */
         if (!job->firstChunk) job->params.fParams.contentSizeFlag = 0;  /* ensure no srcSize control */
-        {   size_t const dictModeError = ZSTD_setCCtxParameter(cctx, ZSTD_p_forceRawDict, 1);  /* Force loading dictionary in "content-only" mode (no header analysis) */
-            size_t const initError = ZSTD_compressBegin_advanced(cctx, job->srcStart, job->dictSize, job->params, job->fullFrameSize);
-            if (ZSTD_isError(initError) || ZSTD_isError(dictModeError)) { job->cSize = initError; goto _endJob; }
-            ZSTD_setCCtxParameter(cctx, ZSTD_p_forceWindow, 1);
+        { ZSTD_CCtx_params jobParams = job->params;
+          /* Force loading dictionary in "content-only" mode (no header analysis) */
+          size_t const dictModeError =
+              ZSTD_CCtxParam_setParameter(&jobParams, ZSTD_p_dictMode, 1);
```

Comment:
I would prefer the use of relevant enum type `ZSTD_dictMode_e` (value `ZSTD_dm_rawContent` for this case).
It's easier to understand than `1`.